assets/src/blocks/HubspotForm/HubspotFormSidebar.js

```diff
@@ -0,0 +1,112 @@
+import { CheckboxControl, PanelBody, PanelRow, Button, TextControl } from '@wordpress/components';
+import { MediaUpload, MediaUploadCheck, InspectorControls } from '@wordpress/block-editor';
+import { URLInput } from '../../components/URLInput/URLInput';
+import { useToAttribute } from './hooks/useToAttribute';
+
+const { __ } = wp.i18n;
+
+const PanelRowWraper = ({ labelText, helpText, children }) => {
```

Comment:
It's not a big deal, but there's a small typo here, it should be `Wrapper` with 2 p's 🙂